Sources/Widgets/Representations/ArrowHandleRepresentation/index.js

```diff
@@ -176,16 +250,22 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
 
 const DEFAULT_VALUES = {
   defaultScale: 1,
+  orientation: [0, 0, 0],
```

Comment:
how about setting a meaningful initial value ?

Reply:
for this i think it is important to give it a neutral 1st value because orientation has a direct link to 2 behaviors, set handle direction when moving a handle, and set handle orientation to face camera. I have a limited understanding of the use of matrices in 3d graphics so i don't see what a meaningful value would be.

Reply:
orientation does not play for "facing the camera".
I think by default, it should go orient an arrow from left to right.